src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/SvgImageSource_Icons.xaml.cs

```diff
@@ -12,17 +12,26 @@
 using Microsoft.UI.Xaml.Controls.Primitives;
 using Microsoft.UI.Xaml.Media;
 using Microsoft.UI.Xaml.Media.Imaging;
+using UITests.Shared.Helpers;
 
 namespace UITests.Windows_UI_Xaml_Controls.ImageTests;
 
 [Sample("Image")]
-public sealed partial class SvgImageSource_Icons : Page
+public sealed partial class SvgImageSource_Icons : Page, IWaitableSample
 {
+	private readonly Task _samplePreparedTask;
+
 	public SvgImageSource_Icons()
 	{
 		this.InitializeComponent();
+		_samplePreparedTask = Task.WhenAll(
+			WaitableSampleImageHelpers.WaitAllImages(image1),
+			WaitableSampleImageHelpers.WaitAllImages((SvgImageSource)imageIcon1.Source)
+		);
```

Comment:
Looks like this is causing a timeout in CI. Feels like the task doesn't complete which may mean that SvgImageSource isn't firing `Open`/`OpenFailed` events